### ISSUE TITLE
x/tools/present: add speaker notes to the title page

### ISSUE THREAD

**Issue by rakyll** [golang/go#17080]
The title page is a good opportunity to give some context and background before starting to talk. Notes would be useful not to miss any critical item.

Also, for those who suffer from stage fright and forget their name, being able to see the speaker notes on the first page would be nice to have :)

/cc @audreylim

**Comment by alexcarol** [comment]
If no one's on it i'd like to take this on.

**Comment by rakyll** [comment]
@alexcarol, please do!

**Comment by alexcarol** [comment]
@rakyll change submitted
https://go-review.googlesource.com/#/c/31396/ 
is this what you were looking for?

**Comment by rakyll** [comment]
Closed via https://github.com/golang/tools/commit/c6efba04dd0d931bb11cd7f556285fa3c9305398.
